src/react/canvas_ux.tsx

```diff
@@ -174,23 +135,16 @@ export function LifeView(props: {
 
 	const handleKeyDown = (e: React.KeyboardEvent) => {
 		if (e.key === "Enter") {
-			props.life.moment.addSession("" + inputValue + " - " + getDateTime());
+			const desc = inputValue;
 			setInputValue("");
+			// ! TODO: try and populate what we have, and have GPTService update it when response comes in
+			GPTService.prompt(desc).then((moments) => {
```

Comment:
`.catch` block for this floating promise?

Reply:
Personal choice, but I feel like the service should catch any error and provide necessary default behavior. If an error ends up at this layer, then something is seriously busted.

Reply:
#nit 

Right. But my thought is, let's not add a dangling rejection to that seriously busted state - at least log console.error or something.